playbooks/roles/streisand-gateway/templates/vhost.j2

```diff
@@ -9,23 +9,43 @@ server {
   ssl_certificate      {{ nginx_self_signed_certificate }};
   ssl_certificate_key  {{ nginx_private_key }};
 {% endif %}
-  add_header Strict-Transport-Security max-age=631138519;
 
   auth_basic            "Authorization Required";
   auth_basic_user_file  {{ streisand_gateway_htpasswd_file }};
 
+  ssl_protocols TLSv1.2;
```

Comment:
I would prefer to see this target Mozilla's "intermediate" profile for `ssl_protocols`, leaving TLS 1.0 and 1.1 enabled for the time being. I don't have a sense of how many HTTPS clients are likely to be affected by dropping TLS 1.0/1.1 and we should be conservative about breaking compatibility.